src/pages/billing/SpendReport/SpendReport.ts

```diff
@@ -5,26 +5,32 @@ import { div, h, span } from 'react-hyperscript-helpers'
 import {
   customSpinnerOverlay,
   IdContainer,
+  Link,
   Select
 } from 'src/components/common'
 import { Ajax } from 'src/libs/ajax'
 import { FormLabel } from 'src/libs/forms'
 import { useCancellation } from 'src/libs/react-utils'
+import * as Utils from 'src/libs/utils'
+import { CloudPlatform } from 'src/pages/billing/models/BillingProject'
 import { CostCard } from 'src/pages/billing/SpendReport/CostCard'
 import { ErrorAlert } from 'src/pages/billing/SpendReport/ErrorAlert'
 
 
 const LazyChart = lazy(() => import('src/components/Chart'))
 const maxWorkspacesInChart = 10
 
-const OtherMessaging = ({ cost }) => {
+const OtherMessaging = ({ cost, cloudPlatform }) => {
   const msg = cost !== null ?
     `Total spend includes ${cost} in other infrastructure or query costs related to the general operations of Terra.` :
-    'Total spend includes infrastructure or query costs related to the general operations of Terra'
+    'Total spend includes infrastructure or query costs related to the general operations of Terra.'
   return div({ 'aria-live': cost !== null ? 'polite' : 'off', 'aria-atomic': true }, [
     span({ 'aria-hidden': true }, ['*']),
     '',
-    msg
+    msg,
+    cloudPlatform !== 'AZURE' ? '' : [' See ',
+      h(Link, { href: 'https://support.terra.bio/hc/en-us/articles/12029087819291-Overview-Costs-and-billing-in-Terra-on-Azure', ...Utils.newTabLinkProps }, ['our documentation']),
```

Comment:
We are supposed to include the "pop out" icon for things that will take you outside of Terra, including our documentation. And therefore it's good if the link appears at the end (with text that would make sense on its own for navigation by link). Perhaps touch base with Lou for wording suggestions.

Reply:
BTW, there is an "ExternalLink" component you can use for the styling. `src/pages/billing/NewBillingProjectWizard/StepWizard/ExternalLink`

Reply:
It could be as simple as making the link text "See documentation for costs overview". Sort of like this--

![image](https://user-images.githubusercontent.com/484484/227281640-7490bf17-8b22-433b-8dcb-85f16268fc52.png)

Reply:
I changed it to "See our documentation to learn more about Azure costs." as a placeholder. I'm not familiar with how this normally works for UI changes, is a small wording question like this something that we can decide on ourselves or do we need to get UX involved?